modules/end-user-guide/partials/proc_mounting-a-secret-as-a-file-into-a-workspace-container.adoc

```diff
@@ -7,23 +7,57 @@
 
 WARNING: On 
 ifeval::["{project-context}" == "che"]
-{platforms-name} older than v1.13 and
+{kubernetes} older than v1.13 and
 endif::[]
-OpenShift {ocp3-ver}, secrets mounted as file overrides volume mounts defined in the devfile.
+OpenShift {ocp3-ver}, secrets mounted as a file override volume mounts defined in the devfile.
 
-This section describes how to mount a secret from the user's {orch-namespace} as a file in single-workspace or multiple-workspace containers of {prod-short}.
+In {prod-short}, you can mount the {platforms-name} secret from your {orch-namespace} as a file in single-workspace or multiple-workspace containers.
 
 .Prerequisites
 
-* A running instance of {prod-short}. To install an instance of {prod-short}, see {link-installing-an-instance}.
+* A running instance of {prod-short}. See {link-installing-an-instance}.
 
 .Procedure
 
-. Create a new {platforms-name} secret in the {platforms-name} {orch-namespace} where a {prod-short} workspace will be created.
-
-* The labels of the secret that is about to be created must match the set of labels configured in `che.workspace.provision.secret.labels` property of {prod-short}. The default labels are:
-
-* `app.kubernetes.io/part-of: che.eclipse.org` 
-* `app.kubernetes.io/component: workspace-secret`:
+. Create a new secret in the {platforms-namespace} where a new workspace is deployed.
+. Match the secret's `labels` to the set of labels configured in the `che.workspace.provision.secret.labels` property of {prod-short}.
+. Define the default behavior for mounting secrets in the `automount-workspace-secret` annotation, which takes a boolean value:
+** `'true'` enables automatic mounting into all workspace containers.
```

Comment:
Does the value really need to be quotes?

Reply:
@rkratky Have a look, these values are in quotes in four examples (possibly written by the same author): https://www.eclipse.org/che/docs/che-7/end-user-guide/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container/ If this doesn't seem right, we can of course double-check with SMEs.

Reply:
I suggest you do check it with an SME because it seems odd. If the quotes don't need to be there, then remove them from the other examples, too.

Reply:
Quote:
`The keys and the values in the map must be strings. In other words, you cannot use numeric, boolean, list or other types for either the keys or the values.`
https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/